### DISCUSSION TITLE
Does Magma's MME supports NB-IoT

### DISCUSSION THREAD

**Discussion by cibero42** [magma/magma#15427] [Q&A]
Hello, I'm part of a research team which is trying to build something on top of NB-IoT utilising only open source software.

We are planning on using Openairinterface's eNB, but we are still searching for an EPC which as NB-IoT support.

Digging OAI's github, we found a not-well-maintained EPC from OAI, which stated that they dropped their MME in favour of Magma's.

Does Magma's MME supports NB-IoT functions? I couldn't find anything related to it in the documentation nor in the release notes - just one issue from 2021 stating that it would be supported in the future.

Thanks in advance!

**Comment by jao-tm** [comment]
Hi,
NB-IoT is not supported
